tests/envs/test_pybullet_robots.py

```diff
@@ -174,7 +177,7 @@ def test_fetch_pybullet_robot():
     assert np.allclose(robot_state, recovered_state, atol=1e-3)
     assert np.allclose(robot.get_joints(),
                        robot.initial_joint_values,
-                       atol=1e-3)
+                       atol=1e-2)
```

Comment:
what happened here?

Reply:
we changed the reset_state() code to first reset to `self._initial_joint_values` and then call IK + reset joint states again, this time to (rx, ry, rz). so there will be larger-than-before errors in this allclose. but i still prefer to keep it this way

Reply:
Kk, fine with me as long as supercloud checks out